src/main/kotlin/no/nav/syfo/fastlege/ws/adresseregister/OrganisasjonPerson.kt

```diff
@@ -0,0 +1,5 @@
+package no.nav.syfo.fastlege.ws.adresseregister
+
+data class OrganisasjonPerson(
```

Comment:
Kan vi gi denne klassen et navn fra vårt domene som gir mer mening for oss? Det er ikke mye OrganisasjonPerson igjen, bare ForeldreenhetHerId. Noen forslag: PraksisInfo, Praksis, LegeKontor, LegeKontorInfo, elns

Reply:
Gode forslag. Jeg valgte PraksisInfo.